### PR TITLE
point to full list of compilation options

### PR DESCRIPTION
**Pull request by ferdymercury**
and clarify some alternatives more explicitly

### CODE REVIEW

**Review by couet — APPROVED** [review]
LGTM

**Review by couet — APPROVED** [review]

**Comment by ferdymercury** [comment]
(This one's ready too from my side)

File: manual/root_macros_and_shared_libraries/index.md [hunk 1]
```diff
@@ -138,7 +156,19 @@ root [0] .L MyScript.C+
 
 The `+` option compiles the code and generates a shared library `MyScript_C.so` (`MyScript_C.dll` on Windows).
 The `+` command rebuilds the library only if the ROOT macro or any of the files it includes
-are newer than the library. To force recompiling the library in any case, use `++`:
+are newer than the library. To force recompiling the library in any case, use `++`.
```
Comment (eguiraud) [inline manual/root_macros_and_shared_libraries/index.md:159]:
```suggestion
are newer than the library. To force a recompilation of the library, use `++`.
```

**Review by couet — APPROVED** [review]
LGTM. If it is fine for @eguiraud (who did some comments), I think we can merge.

**Review by couet — APPROVED** [review]

**Review by eguiraud — APPROVED** [review]

**Comment by ferdymercury** [comment]
> LGTM. If it is fine for @eguiraud (who did some comments), I think we can merge.

egiraud approved, I think this one is ready to merge. Thanks!

**Pull request merged**